src/chunkserver/network_worker_thread.cc

```diff
@@ -176,103 +181,101 @@ void worker_create_attached_packet(ChunkserverEntry *eptr,
 	outpacket->bytesLeft = outpacket->packet.size();
 	outpacket->startPtr = outpacket->packet.data();
 
-	eptr->attachPacket(std::move(outpacket));
+	attachPacket(std::move(outpacket));
 }
 
-uint8_t *worker_create_attached_packet(ChunkserverEntry *eptr, uint32_t type,
-                                       uint32_t size) {
+uint8_t *ChunkserverEntry::createAttachedPacket(uint32_t type,
+                                                uint32_t opSize) {
```

Comment:
```suggestion
                                                uint32_t outPacketSize) {
```
Consider full descriptive naming for this variable.

Reply:
Applied